preprocessing/lib/name-corrections.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * A global list of name corrections for division/district/upazila/union spelling variations.
```

Comment:
this file isn't a global list, it contains one but shouldn't:
* [ ] move the correction to the file
* [ ] clarify this .js file loads the name corrections

Reply:
Resolved in 798f8cf